src/main/java/org/archive/io/arc/ARCRecord.java

```diff
@@ -263,6 +264,7 @@ private ArchiveRecordHeader parseHeaders(final InputStream in,
             bodyOffset += getTokenizedHeaderLine(in, secondLineValues);
             version = ((String)secondLineValues.get(0) +
                 "." + (String)secondLineValues.get(1));
+            origin = (String)secondLineValues.get(2);
```

Comment:
I don't know if it is safe to assume it is there, but I do know that that field will frequently contain erroneous information.

I just had a look at our own ARCs and they all have `InternetArchive` set for this "origin". Seems webarchive-commons (and thus Heritrix) ARCWriter just hardcodes that value: https://github.com/jrwiebe/webarchive-commons/blob/master/src/main/java/org/archive/io/arc/ARCWriter.java#L272

Given how prevalent this abuse is by now, I think it is safe to say that this field has zero informational value.